db/db_impl/db_impl.cc

```diff
@@ -1496,6 +1496,29 @@ bool DBImpl::SetPreserveDeletesSequenceNumber(SequenceNumber seqnum) {
   }
 }
 
+Status DBImpl::IncreaseFullHistoryTsLow(ColumnFamilyHandle* column_family,
+                                        std::string& ts_low) {
+  if (column_family == nullptr) {
+    return Status::InvalidArgument("column family handle is null");
```

Comment:
Handling a null column family handle ptr should be consistent.
In `GetFullHistoryTsLow()`, null cf handle indicates default column family, but here it's an error.

Reply:
fixed